.github/workflows/build-docker.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Setup Node 16.x
         uses: actions/setup-node@v2.4.1
         with:
-          node-version: '16.0.0'
+          node-version: '16.13.0'
```

Comment:
Are we able to do a more generic `16.13` and have it pick up the latest minor in that series, or does this need a very specific version like this?

Reply:
yeah, it would, that would be better.  Will make the change

Reply:
resolved with latest commit